openstack/ironic/templates/etc/_ironic.conf.tpl

```diff
@@ -120,21 +120,20 @@ port_setup_delay = {{ .Values.neutron_port_setup_delay }}
 [oslo_middleware]
 enable_proxy_headers_parsing = True
 
-{{- if .Values.watcher.enabled }}
+{{ if .Values.watcher.enabled -}}
```

Comment:
Generally, I have made good experiences with `{{-` practically everywhere.

Reply:
But here it would remove the empty line, so the result looks like this:

```
[oslo_middleware]
enable_proxy_headers_parsing = True
[watcher]
enabled = true
service_type = baremetal
config_file = /etc/ironic/watcher.yaml
```

instead of 

```
[oslo_middleware]
enable_proxy_headers_parsing = True

[watcher]
enabled = true
service_type = baremetal
config_file = /etc/ironic/watcher.yaml
``` 

The first block is from a live pod, so that's the current state, and from my understanding it should more look like the second example. From my reading of https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace that should do that and local templating confirms it.

Reply:
You can add the empty line to the block you include...
```
{{- if something someting }}

[block]
{{-  end }}
```